cardano-api/src/Cardano/Api/TxBody.hs

```diff
@@ -717,9 +717,9 @@ data TxBody era where
 
           -- The 'Ledger.Metadata' is really the /auxiliary/ data, it consists
           -- of one or several things, depending on era:
-          -- * tranaction metadata   (in Shelley and later)
-          -- * auxiliary scripts     (in Allegra and later)
-          -- * auxiliary script data (in Allonzo and later)
+          -- • tranaction metadata   (in Shelley and later)
```

Comment:
Might as well fix the typo while at it:
```suggestion
          -- • transaction metadata   (in Shelley and later)
```
Also, these bullets are unicode. That's fine for me, how do others feel? An ASCII alternative would be `-` or `o`.

Reply:
The character is up to you, as long as it's not `*`, `|`, or `^`.